src/input/keyboard_and_mouse.rs

```diff
@@ -24,6 +26,11 @@ impl KeyboardAndMouse {
         self.cursor_position
     }
 
+    /// Returns the mouse wheel movements during the last interaction.
+    pub fn mouse_wheel(&self) -> Point {
```

Comment:
Oh, right. About the mouse wheel. I would probably implement a `WheelMovement` struct or similar and return that. A `Point` has a different kind of connotation.

Reply:
Struct created please review :smiley: